<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color:qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0.983, stop:0 rgba(0, 255, 255, 255), stop:0.0170455 rgba(29, 59, 118, 255), stop:1 rgba(255, 255, 255, 255));}
</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 36pt &quot;MS Shell Dlg 2&quot;;
font: 48pt &quot;Poor Richard&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>WELCOME</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>521</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;Viner Hand ITC&quot;;</string>
    </property>
    <property name="text">
     <string>Excellent Service from APEX Filling Station, Kegalle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cust_Button">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(25, 76, 76);
</string>
    </property>
    <property name="text">
     <string>Customer Portal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="station_Button">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>390</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(25, 76, 76);</string>
    </property>
    <property name="text">
     <string>Fuel Station Portal </string>
    </property>
   </widget>
   <widget class="QPushButton" name="register_Button">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>480</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(25, 76, 76);</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
